assets/js/toggle-all.js

```diff
@@ -0,0 +1,15 @@
+document.querySelector("#toggle-on").addEventListener("click", function(event) {
+  var checkboxes = document.querySelectorAll('input[type="checkbox"]');
```

Comment:
Rather than selecting _all_ checkboxes on the page (which seems like it could inadvertently target some sort of header control or route switcher or something we add in the future), it would be good to scope it to just the checkboxes we care about. I think we should wrap the `for` loop that iterates and creates the toggles in a class `js-sign-toggles` (I think dotcom used the convention that HTML elements controlled via JS had a `js-` prefix, so we wouldn't accidentally change it), and then update the selector here to be `'.js-sign-toggles input[type="checkbox"]'`.

Reply:
fair enough - pr to follow